decimalSuite_test.go

```diff
@@ -41,6 +43,9 @@ var tests = []string{"",
 	// 	"dectest/ddMinMag.decTest",
 	// 	"dectest/ddMinus.decTest",
 }
+var supportedRounding = []string{"half_up",
+	"half_even"}
```

Comment:
Unnecessary wrapping. If you're going to wrap, put each element and the terminating `}` on separate lines.